README.md

```diff
@@ -925,6 +925,59 @@ Back endpoint responsible for fetching and sending information to the database r
 `api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not: true = symptomatic, false = not symptomatic.
 
 
+### Jobs
+Back endpoint responsible for fetching and sending information to the database regarding Jobs and timesheet used to clock shifts.
```

Comment:
While this follows the example of Wellness Check, all the entries before this summer have a style that starts with "What is it?"  Please follow that style.  (Fixing Wellness would be a bonus :-)

Reply:
I see that makes sense. Thanks.